communities.json

```diff
@@ -722,6 +722,13 @@
       "inviteCode": "scala",
       "githubUrl": "https://github.com/scala"
     },
+    {
+      "logo": "svelte.svg",
+      "title": "Svelte",
+      "quote": "We use Discord to communicate with the wider community, answer questions as well as organise events.",
+      "inviteCode": "svelte",
```

Comment:
This doesn't appear to be a current invite code, I need one in order to find your server

Reply:
```suggestion
      "inviteCode": "sJ5JkuRHWu",
```